### ISSUE TITLE
DeclareAsNullable does not trigger for optional parameters

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn#43346]
**Version Used**: 

Version 16.7.0 Preview 1.0 [30014.17.master]

**Steps to Reproduce**:

1. Create a new .NET Core Console App and add `<Nullable>enable</Nullable>` to the project file
2. Replace code in Program.cs with this
```csharp
class Program {
    void Method(string args = null) {
    }
}

```
3. Invoke Lightbuld on the `null` keyword in the optional parameter

**Expected Behavior**:
A Codefix to add `?` as added in [PR #40753](https://github.com/dotnet/roslyn/pull/40753)

**Actual Behavior**:
DeclareAsNullable does not trigger
![image](https://user-images.githubusercontent.com/9797472/79261314-80ae3d80-7e44-11ea-9092-a50b88214c60.png)

NOTE:
This codefix does trigger for return types

![image](https://user-images.githubusercontent.com/9797472/79261443-adfaeb80-7e44-11ea-94d0-c033e016d16c.png)

**Comment by Youssef1313** [comment]
@jmarolf I can't reproduce in SharpLab.

![image](https://user-images.githubusercontent.com/31348972/95026744-dc3ee280-0693-11eb-9f54-211bd2fe9954.png)

**Comment by jmarolf** [comment]
yep fixes :)
